scripts/provisioning-tests

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -x
+
+cd $(dirname $0)/..
+
+METADATA_DIR=/var/lib/rancher-data/driver-metadata
+
+mkdir -p $METADATA_DIR
+
+cp ./data/data.json $METADATA_DIR/data.json
+
+if [[ -n DIST ]]; then
```

Comment:
Hmm, where is the env var  `DIST` set? 
Also, do you want to overwrite its value to `k3s` when it is set? I feel what we need is the opposite. Can you confirm?